jib-core/src/main/java/com/google/cloud/tools/jib/api/JavaContainerBuilder.java

```diff
@@ -124,7 +125,8 @@ public static JavaContainerBuilder fromDistroless() {
    * @see <a href="https://github.com/GoogleContainerTools/distroless">The distroless repository</a>
    * @deprecated Use {@code from()} with the image reference {@code gcr.io/distroless/java/jetty}
    *     and change the app root by calling {@code
-   *     JavaContainerBuilder.setAppRoot("/jetty/webapps/ROOT")}.
+   *     JavaContainerBuilder.setAppRoot("/jetty/webapps/ROOT")}. However, using Distroless Java is
+   *     no longer recommended.
```

Comment:
Does this need to be?:
```suggestion
   *     JavaContainerBuilder.setAppRoot("var/lib/jetty/webapps/ROOT")}. However, using Distroless Java is
   *     no longer recommended.
```

Reply:
Sharp eye. The answer is no. When using Distroless, the app root should be `/jetty/webapps`.

Reply:
Oh right, oops. I missed the part where distroless was mentioned haha